### DISCUSSION TITLE
Thank you so much

### DISCUSSION THREAD

**Discussion by IlluminatorX** [audiamus/AaxAudioConverter#189] [General]
A thank you discussions was missing, so...

In my oppinion this is is a great piece of sw. Both the downloader and the converter. I freaking rummaged through the depths of internet and after a long yourney I found these SWs as solution! Thank you, Muchas  gracias etc ;) !

**Comment by Linkinsoldier** [comment]
Just tested it and this software works great - thank you! Audible got a customer they would not have otherwise and I only use it for my right to play my payed for audiobooks on any device I'd like... so everybody is happy :)

I read that you don't want donations and we should "just spread the word" --> so I did. 

THANK YOU @audiamus!
